erpnext/patches/v14_0/discount_accounting_separation.py

```diff
@@ -0,0 +1,8 @@
+import frappe
+
+def execute():
+    doc = frappe.get_doc("Accounts Settings")
+    discount_account = doc.enable_discount_accounting
+    if discount_account:
+        for doctype in ["Buying Settings", "Selling Settings"]:
```

Comment:
```suggestion
        for doctype in ["Buying Settings", "Selling Settings"]:
```

We use tabs for formatting code. 

Setup pre-commit and add editorconfig plugin to automate this: https://github.com/frappe/erpnext/wiki/Pull-Request-Checklist#linting